docusaurus/docs/dev-docs/migration/v4-to-v5/breaking-changes/draft-and-publish-always-enabled.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Draft & Publish always enabled in v5
+description: Draft & Publish is always enabled in Strapi v5 and this is reflected in the Content API models.
+sidebar_label: Draft & Publish always enabled
+displayed_sidebar: devDocsMigrationV5Sidebar
+tags:
+ - breaking changes
+ - content API
+ - draft & publish
+---
+
+import Intro from '/docs/snippets/breaking-change-page-intro.md'
+import MigrationIntro from '/docs/snippets/breaking-change-page-migration-intro.md'
+
+# Draft & Publish is always enabled in Strapi v5
+
+In Strapi v5, the Draft & Publish feature is always enabled. <Intro />
+
+## Breaking change description
+
+<SideBySideContainer>
+
+<SideBySideColumn>
+
+**In Strapi v4**
+
+Content types had a `draftAndPublish` option in their schema. The database schema and admin panel interface changed depending on that option.
+
+</SideBySideColumn>
+
+<SideBySideColumn>
+
+**In Strapi v5**
+
+All content-types now have Draft & publish enabled in the database and in the admin panel interface. The admin panel interface also will be changing.
+
+</SideBySideColumn>
+
+</SideBySideContainer>
+
+## Migration
+
+<MigrationIntro />
+
+### Notes
+
+<!-- TODO: add codemod link -->
+<!-- TODO: add link to strapi upgrade CLI tool documentation -->
+- Strapi provides a [codemod](#) to automatically update all the content-type schemas that are detected by the [`strapi upgrade` tool](#).
```

Comment:
```suggestion
- Strapi provides a [codemods](#) to automatically update all the content-type schemas that are detected by the [`strapi upgrade` tool](#).
```

Reply:
Thanks! I didn't know codemods is always plural.